### PR TITLE
Bump serving to v1.7

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

**Comment by nak3** [comment]
Hmm... only serving bumps is difficult as Reto mentioned in his PR.

**Comment by nak3** [comment]
/test 4.11-operator-e2e-aws-ocp-411

`TestKafkaSourceToKnativeService`

**Comment by ReToCode** [comment]
/test 4.11-operator-e2e-aws-ocp-411

`TestKafkaSourceToKnativeService`

File: openshift-knative-operator/cmd/operator/kodata/knative-serving/1.7.0/1-serving-crds.yaml [hunk 1]
```diff
@@ -590,9 +590,6 @@ spec:
                                 description: 'SecurityContext defines the security options the container should be run with. If set, the fields of SecurityContext override the equivalent fields of PodSecurityContext. More info: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/'
                                 type: object
                                 properties:
-                                  allowPrivilegeEscalation:
```
Comment (skonto) [inline openshift-knative-operator/cmd/operator/kodata/knative-serving/1.7.0/1-serving-crds.yaml:593]: This part is missing?

Comment (nak3) [reply openshift-knative-operator/cmd/operator/kodata/knative-serving/1.7.0/1-serving-crds.yaml:593]: Hmm... it seems we missed https://github.com/openshift-knative/serving/pull/18 for 1.7.

**Pull request merged**